tests/System.Data.SQLite.Tests/SqliteCommandTests.cs

```diff
@@ -9,6 +9,8 @@ public class SqliteCommandTests
 		[SetUp]
 		public void SetUp()
 		{
+			TestUtility.ForceLoggingInitialization();
```

Comment:
Could we use [`SetUpFixture`](https://docs.nunit.org/articles/nunit/writing-tests/attributes/setupfixture.html) instead of repeating this for all test fixtures?

Reply:
I guess.

Reply:
Done.